specification/batch/resource-manager/Microsoft.Batch/stable/2020-03-01/BatchManagement.json

```diff
@@ -4975,7 +4975,7 @@
       "in": "path",
       "required": true,
       "type": "string",
-      "pattern": "^[a-zA-Z0-9]+$",
+      "pattern": "^[a-z0-9]+$",
```

Comment:
>^[a-z0-9]+$ [](start = 18, length = 11)

This parameter is used in resource names, which are supposed to be case-insensitive. This would make those resource names only support lower case and numbers, which is both a violation of the RPC and a breaking change.